README.md

```diff
@@ -34,7 +34,7 @@ It is a simple web app built with React, to demonstrate a project setup involvin
 <details>
   <summary>Server</summary>
   <ul>
-    <li>Ruby-on-Rails [Back-end](https://github.com/adamilare/hello-react-back-end)</li>
+    <li>Ruby-on-Rails [Back-end]https://github.com/adamilare/hello-rails-back-end)</li>
```

Comment:
- [Optional ] The above link redirect to the `dev` branch Kindly make sure that you redirect the the `feat-server `branch.